ansible/prometheus-snapshot.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- hosts: monitoring
+  tasks:
+    - name: Install awscli
+      become: yes
+      ansible.builtin.package:
+        name:
+          - awscli
+
+    - name: Take snapshot
+      ansible.builtin.uri:
+        url: http://localhost:9090/api/v1/admin/tsdb/snapshot
+        method: POST
+        status_code: 200
+      register: _snapshot_res
+
+    - name: Snapshot name
+      debug:
+        var: _snapshot_res.json.data.name
+
+    - name: Archive snapshot
+      community.general.archive:
+        path: "/opt/monitoring/data/snapshots/{{ _snapshot_res.json.data.name }}"
+        dest: "/opt/monitoring/{{ _snapshot_res.json.data.name }}.tar.bz2"
+        format: bz2
+
+  # todo: upload to s3
```

Comment:
may be just download it to control host for now

Reply:
I'll try to bundle a ready-to-go docker-compose.yaml + dashboards, so we could just unpack and run for easier inspection.